### PR TITLE
List

### PR DESCRIPTION
**Pull request by pariyar07**

### CODE REVIEW

File: pages/list.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>WALDO | List</title>
+    <link rel="stylesheet" href="../styles/main.css">
+</head>
+
+<body>
+    <nav class="navbar-content">
+        <img class="nav-icon" src="/assets/images/logo-cl.png" alt="">
+        <div class="nav-list">
+            <li><a href="../index.html">Home</a></li>
```
Comment (sanjitsarkar) [inline pages/list.html:16]: Use ul instead of div as it is semantically correct

File: pages/list.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+                <a class="nav-link" href="./grid.html">Grid</a>
+            </li>
+            <li style="padding-left: 1rem; font-size: 1.5rem; background: #171717; border-radius: 8px;">
+                <a class="nav-link" href="#">List</a>
+            </li>
+        </ul>
+    </aside>
+    <main id="main-doc">
+        <div class="alert main-container">
+            <header>LIST</header>
```
Comment (sanjitsarkar) [inline pages/list.html]: Header tag is not used for adding title or text

File: styles/grid.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
+.grid-container{
+    display: grid;
+    width: 900px;
```
Comment (sanjitsarkar) [inline styles/grid.css:3]: Don't use px for larger values

**Review by sanjitsarkar — APPROVED** [review]

**Review by dubeyaditya29 — COMMENTED** [review]
The code looks clean apart from the suggestions made by @sanjitsarkar good to merge

File: styles/list.css [hunk 4]
```diff
@@ -0,0 +1,42 @@
+.list-container {
+    background-color: var(--secondary-bg);
+    width: 700px;
```
Comment (psychoactiv) [inline styles/list.css:3]: Rem value is preferred else it looks good to merge

**Review by psychoactiv — APPROVED** [review]

File: pages/list.html (hunk 2 above)
Comment (pariyar07) [reply pages/list.html]: Yup thanks. Corrected

**Pull request merged**